### PR TITLE
Coco mask eval

### PR DESCRIPTION
**Pull request by enricoliscio**
Working COCO segmentation evaluation.

### CODE REVIEW

File: keras_maskrcnn/bin/evaluate_coco.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+from keras_retinanet.utils.keras_version import check_keras_version
+
+def get_session():
+    config = tf.ConfigProto()
+    config.gpu_options.allow_growth = True
+    return tf.Session(config=config)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(description='Simple training script for COCO object detection.')
```
Comment (hgaiser) [inline keras_maskrcnn/bin/evaluate_coco.py]: Hmm this is probably also wrong in retinanet, but let's correct it here first of all ;)

File: keras_maskrcnn/losses.py [hunk 2]
```diff
@@ -66,10 +66,10 @@ def _mask(y_true, y_pred):
         masks = keras_retinanet.backend.gather_nd(masks, argmax_overlaps_inds)
 
         # compute mask loss
-        mask_loss = masks - masks_target
-        mask_loss = keras.backend.abs(mask_loss)
-        divisor = keras.backend.shape(masks)[0] * keras.backend.shape(masks)[1] * keras.backend.shape(masks)[2]
-        mask_loss = keras.backend.sum(mask_loss) / keras.backend.maximum(keras.backend.cast(divisor, keras.backend.floatx()), 1)
+        mask_loss  = keras.backend.binary_crossentropy(masks_target, masks)
```
Comment (hgaiser) [inline keras_maskrcnn/losses.py]: This is from the other PR ?

File: keras_maskrcnn/utils/coco_eval.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    results = []
+    image_ids = []
+    for index in range(10): #generator.size()):
+        image = generator.load_image(index)
+        image_shape = image.shape
+        image = generator.preprocess_image(image)
+        image, scale = generator.resize_image(image)
+
+        # run network
+        _, _, _, boxes, classification, masks = model.predict_on_batch(np.expand_dims(image, axis=0))
```
Comment (hgaiser) [inline keras_maskrcnn/utils/coco_eval.py]: Please rename classification to nms_classification. I'm doing the same in retinanet.

**Comment by hgaiser** [comment]
Merci.
